backend/dataall/modules/dataset_sharing/services/share_processors/s3_access_point_process_share.py

```diff
@@ -10,17 +10,18 @@
 log = logging.getLogger(__name__)
 
 
-class ProcessS3Share(S3ShareManager):
+class ProcessS3AccessPointShare(S3AccessPointShareManager):
     def __init__(
-        self,
-        session,
-        dataset: Dataset,
-        share: ShareObject,
-        share_folder: DatasetStorageLocation,
-        source_environment: Environment,
-        target_environment: Environment,
-        source_env_group: EnvironmentGroup,
-        env_group: EnvironmentGroup,
+            self,
```

Comment:
identation seems to be wrong, also below

Reply:
Fixed this in next commit.